### DISCUSSION TITLE
Requesting Inpainting Tweak

### DISCUSSION THREAD

**Discussion by icequake1969** [glucauze/sd-webui-faceswaplab#99] [Ideas]
I use faceswaplab mostly with the inpaint  mask-only function.   It would be nice to ONLY consider faces in the inpaint mask when specifying target faces for swap.    The roop extension worked as such.   But with the current build, it can be difficult and time-consuming  to find the correct face number for a simple one-face selected mask;   particularly if it is a scene with lots of faces in it.    Since you are only wanting to affect the masked area; I don't see any reason it would need to consider faces outside of the mask.    So yes, it would be great to start the target face count in the mask at 0.

Other than that, excellent extension.   You went above and beyond the roop extension.

**Comment by glucauze** [comment]
Yes, i know it's annoying. I might correct that if i have some time. It's not as easy as it seems.
